source/news/2019-10-03.html.markdown

```diff
@@ -4,13 +4,13 @@ date: 2019-10-03
 author: flash-gordon
 ---
 
-Today we add another gem to the toolset making supercharged with dry-effects!
+Today we're introducing another gem and supercharging our toolset: say hello to dry-effects!
```

Comment:
@flash-gordon I wasn't _100%_ of the meaning you were meant to convey in this opening sentence, but it didn't read smoothly to me, so I just took a stab along similar lines. Please do feel free to adjust this if it doesn't fit with what you meant!